### PR TITLE
Tax years

### PR DESCRIPTION
**Pull request by MatthewAlexanderBeck**
Created dynamic tax years radio buttons and resolved failed tests

### CODE REVIEW

File: app/models/SelectTaxYear.scala [hunk 1]
```diff
@@ -22,12 +22,12 @@ sealed trait SelectTaxYear
 
 object SelectTaxYear extends Enumerable[SelectTaxYear] {
 
-  case object Option1 extends WithName("option1") with SelectTaxYear
-  case object Option2 extends WithName("option2") with SelectTaxYear
-  case object Option3 extends WithName("option3") with SelectTaxYear
-  case object Option4 extends WithName("option4") with SelectTaxYear
+  case object CY extends WithName("currentYear") with SelectTaxYear
```
Comment (oliverslade) [inline app/models/SelectTaxYear.scala]: Can we rename these to something a bit more descriptive. It wont be CY it should be CY-2 to 6

File: conf/messages.en [hunk 2]
```diff
@@ -167,10 +167,10 @@ fullOrPartialClaim.blank = Select whether you want to claim all some of the amou
 
 selectTaxYear.title = Which tax year does your claim apply to?
 selectTaxYear.heading = Which tax year does your claim apply to?
-selectTaxYear.option1 = 6 April 2017 to 5 April 2018
-selectTaxYear.option2 = 6 April 2016 to 5 April 2017
-selectTaxYear.option3 = 6 April 2015 to 5 April 2016
-selectTaxYear.option4 = 6 April 2014 to 5 April 2015
+selectTaxYear.currentYear = {0} to {1}
```
Comment (oliverslade) [inline conf/messages.en]: again with the naming of these

File: .g8/optionsPage/app/forms/$className$Form.scala [hunk 3]
```diff
@@ -3,7 +3,7 @@ package forms
 import play.api.data.Form
 import play.api.data.Forms._
 import play.api.data.format.Formatter
-import utils.RadioOption
+import utils.RadioOptionDynamic
```
Comment (oliverslade) [inline .g8/optionsPage/app/forms/$className$Form.scala]: I think this is a hangover from our tomfoolery

**Comment by MatthewAlexanderBeck** [comment]
@oliverslade changes applied, ready for review

**Pull request merged**